### PR TITLE
RHCHI message processor, and tests.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_umb/rhchi.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class RHCHIProcessor(BaseProcessor):
+    topic_prefix_re = r'/topic/VirtualTopic\.eng'
+
+    __name__ = 'rhchi'
+    __description__ = 'the RHCHI Container Validation Pipeline'
+    __link__ = 'https://chi-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/'
+    __docs__ = 'https://docs.engineering.redhat.com/display/RHAPPINFRA/Container+E2E+QE+-+Full+Workflow'
+    __obj__ = 'pipeline progress'
```
Comment (mprahl) [inline fedmsg_meta_umb/rhchi.py]: The other processor's seem to use capitals for the first letter of each word. Not sure where this data gets displayed.

Comment (mprahl) [inline fedmsg_meta_umb/rhchi.py]: Also, should this be more specific? Pipeline is a very vague term.

Comment (ralphbean) [inline fedmsg_meta_umb/rhchi.py]: Yes, will adjust.  Thanks!

(It technically only gets displayed in the configuration panel of the [fedmsg-notify](https://github.com/fedora-infra/fedmsg-notify) tool, which doesn't matter too much for us.)

**Comment by mprahl** [comment]
I'm not too familiar with the codebase, but it looks good to me. If you're comfortable with it, feel free to merge.

**Comment by ralphbean** [comment]
Thanks @mprahl!

**Pull request merged**